### DISCUSSION TITLE
Is massive executions of getSignatureStatuses normal?, it does not allow me to open phantom on some occasions

### DISCUSSION THREAD

**Discussion by iamsteeve** [phantom/docs#124] [Help & Support] [answered]
### Summary

Is massive executions of getSignatureStatuses normal? It does not allow me to open phantom on some occasions, on some occasions my phantom wallet makes thousands of requests through a phishing.f2bd544b.js file in the image it only made a few but sometimes these are Thousands of requests come back and break the app or take a while to initialize.

<img width="1613" alt="Captura de pantalla 2023-11-22 a la(s) 9 26 14" src="https://github.com/phantom/docs/assets/33048029/ec7f0bf2-a528-4801-b4c1-bc69d61c84d1">

### Steps to Reproduce

<img width="1677" alt="Captura de pantalla 2023-11-22 a la(s) 5 24 52" src="https://github.com/phantom/docs/assets/33048029/10165b87-c96b-45ee-9fc6-8f82b1ec1d11">
When I inspect the wallet I notice the requests.

### Phantom Version

23.17.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@iamsteeve Thank you for writing in regarding this! Would you mind providing a the URL to a dapp where you are seeing this please?
